.github/workflows/create-tag.yml

```diff
@@ -0,0 +1,34 @@
+name: db-syncer -Create Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: "The branch to checkout when creating the image."
+        required: true
+        default: "release_pipeline"
+      
+env:
+  GITHUB_TOKEN: ${{ secrets.MY_TOKEN }}   
+  release-version: 0.0.0
```

Comment:
This will always be `0.0.0`, we can just call `${{ github.event.inputs.ref }}` directly instead of creating this env var.

I think it's also possible todo `${{ inputs.ref }}`